src/PlaywrightSharp/Playwright.cs

```diff
@@ -23,6 +23,11 @@ public sealed class Playwright : ChannelOwnerBase, IPlaywright, IChannelOwner<Pl
         /// </summary>
         public const int DefaultTimeout = 30_000;
 
+        /// <summary>
+        /// Environment variable use to set the browsers path.
+        /// </summary>
+        public const string BrowsersPathEnvironmentVariable = "PLAYWRIGHT_BROWSERS_PATH";
```

Comment:
Is there any nicer place to put this? I'm just nit-picking here, but having a `const` feels _dirty_ to me.

Reply:
Why does it feel dirty?
These constants are mapping constants in upstream.

Reply:
That's fine then, let's leave it where it is. Maybe as we grow & add more env variables we can have a struct or something to _separate_/group them together somewhere.